src/drawer.js

```diff
@@ -383,6 +384,9 @@ $.Drawer.prototype = /** @lends OpenSeadragon.Drawer.prototype */{
 
         this.context.save();
         this.context.globalAlpha = opacity;
+        if (compositeOperation !== undefined && compositeOperation !== null) {
```

Comment:
This should probably just check to see if it's truthy: `if (compositeOperation)`